modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_sla-card.scss

```
@@ -0,0 +1,49 @@
+.sla-container:hover {
+	.btn {
+		background-color: white;
+		border-color: rgba(11, 95, 255, 0.25);
+		box-shadow: $elevation-fluffy-1;
+		display: inline-block !important;
+		padding: 0.25rem;
+		transform: translate(12rem, -3rem);
+		width: 2.1rem;
+		z-index: 100;
+	}
+	.btn:hover {
+		border-color: white;
+	}
+}
+.card {
+	box-shadow: $elevation-fluffy-1;
+}
+.sla-card-holder {
+	position: relative;
+}
+.sla-card-holder .sla-card {
+	filter: blur(2px);
+	position: absolute;
+	transition: all 0.4s;
+}
+.sla-multiple-card .sla-card-active {
+	filter: blur(0);
+	transform: translate(-1rem, 1rem);
+	transition: all 0.4s;
+	z-index: 100;
+}
+.sla-platinum {
+	background-color: #f9fafb;
+	border: 1px solid #e1e1e4;
+}
+.sla-gold {
+	border: 1px solid var(--color-brand-secondary-lighten-4);
+}
+.sla-limited {
+	border: 1px solid #0b5fff99;
+}
+.sla-label {
+	font-size: 0.75rem;
+}
+.n-sla-card {
+	height: 5rem;
+	width: 12rem;
+}
```

Custom colors should be present in style-book

```suggestion
.cp-sla-container:hover {
	.btn {
		background-color: white;
		border-color: rgba(11, 95, 255, 0.25);
		box-shadow: $elevation-fluffy-1;
		display: inline-block !important;
		padding: 0.25rem;
		transform: translate(12rem, -3rem);
		width: 2.1rem;
		z-index: 100;
	}
	
	.btn:hover {
		border-color: white;
	}
}

.cp-card {
	box-shadow: $elevation-fluffy-1;
}

.cp-sla-card-holder {
	position: relative;
}

.cp-sla-card-holder .cp-sla-card {
	filter: blur(2px);
	position: absolute;
	transition: all 0.4s;
}

.cp-sla-multiple-card .cp-sla-card-active {
	filter: blur(0);
	transform: translate(-1rem, 1rem);
	transition: all 0.4s;
	z-index: 100;
}

.cp-sla-platinum {
	background-color: #f9fafb;
	border: 1px solid #e1e1e4;
}

.cp-sla-gold {
	border: 1px solid var(--color-brand-secondary-lighten-4);
}

.cp-sla-limited {
	border: 1px solid #0b5fff99;
}

.cp-sla-label {
	font-size: 0.75rem;
}

.cp-n-sla-card {
	height: 5rem;
	width: 12rem;
}
```

---

Nice!